packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2178,7 +2217,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     }
 
     private canSendOps() {
-        return this.connected && !this.deltaManager.readOnlyInfo.readonly;
+        return this.connected && !this.deltaManager.readOnlyInfo.readonly && !this.delayConnectedP;
```

Comment:
we should never be in a state where connected == true and delayConnectedP !== undefined, right?